azure-iot-device/tests/common/test_mqtt_transport.py

```diff
@@ -292,10 +294,34 @@ def test_use_no_password(self, mocker, mock_mqtt_client, transport):
         ],
     )
     def test_calls_paho_connect(self, mocker, mock_mqtt_client, transport, password):
+
+        mqtt_transport.DEFAULT_KEEPALIVE = fake_keepalive
+
         transport.connect(password)
 
         assert mock_mqtt_client.connect.call_count == 1
-        assert mock_mqtt_client.connect.call_args == mocker.call(host=fake_hostname, port=8883)
+        assert mock_mqtt_client.connect.call_args == mocker.call(
+            host=fake_hostname, port=8883, keepalive=mocker.ANY
```

Comment:
>keepalive=mocker.ANY [](start = 43, length = 20)

should this be fake_keepalive ?

Reply:
That depends on the order the tests run.  If this test runs after the one that changes it to fake_keepalive, then you're right.  If it runs before that test, then you're wrong.  This test doesn't really care what the value is.